### ISSUE TITLE
Vue version of ui lacks css (theme) for element-ui components such as table spinner

### ISSUE THREAD

**Issue by natesnapp** [creativetimofficial/ct-vue-now-ui-dashboard-pro#9]
It appears that the loading svg animation for el-table element is sized inappropriately large and is not doing animation. I have a simple test which I have attached and made as a routable content page. This is using the same code from this this fiddle which has a css applied (unpkg.com/element-ui@2.3.4/lib/theme-chalk/index.css).

See example:
TestTableSpinner.txt

And here is what appears when rendered (image shows an additional disabled button and spinner from previous version of the code.. TestTableSpinner.txt only has bare minimum needed):

![image](https://user-images.githubusercontent.com/2713065/38834024-05c441f2-417c-11e8-843d-e4dfc0f58917.png)

I was able to fix this by adding the element ui chalk theme, which I first attempted by importing via package.json as in the below snippet.

  "dependencies": {
    "axios": "^0.18.0",
    "bootstrap": "^4.0.0",
    "chart.js": "^2.7.1",
    "element-theme-chalk": "^2.3.4",

I then added this to the now-ui-dashboard.scss but it failed to import
// element-ui theme
@import 'element-theme-chalk';

so I commented out and imported externally:
@import url("//unpkg.com/element-ui@2.3.4/lib/theme-chalk/index.css");

which worked. Would like to have 1) a better way to apply themes and doc showing how themes are to be used, 2) for this specific issue, would like to have it fixed so I can import the package.json theme and not require external import which is disallowed by our company policy.

Thanks!

**Comment by cristijora** [comment]
Hi @natesnapp 
The initial problem is that element-ui css was not imported and therefore the spinner didn't render correctly.
Usually when people use mac, they miss the `.babelrc` file from the archive because it's hidden on mac and therefore don't copy it in their projects.

The babelrc file should contain a plugin which imports element-ui css like this
```
"plugins": [
    [
      "component",
      {
        "libraryName": "element-ui",
        "styleLibraryName": "theme-chalk"
      }
    ]
  ]
```

This is necessary so in the end only the components you import (e.g `import { Table } from 'element-ui'` are imported in the final bundle rather than the whole library.

Long story short, if you want to customize the element ui css colors, they have a guide for that here http://element.eleme.io/#/en-US/component/custom-theme 
I would recommend going with this option http://element.eleme.io/#/en-US/component/custom-theme#cli-theme-tool 
Follow the guide there and define the variables from element to the ones you need. For simplicity you could import the `variables` from Vue Now UI Dashboard and reference them directly 

```
@import "path_to_dashboard_variables"
$--color-primary: $primary-color;
```

Hope this helps and let me know if you need further help.

**Comment by cristijora** [comment]
Closing due to inactivity
